src/style.css

```diff
@@ -39,6 +39,8 @@ a {
 #root {
   overflow: auto;
   height: 100%;
+  position: relative;
+  z-index: 0;
```

Comment:
Do we need the z-index?

Reply:
It is necessary to put a z-index on the root in order to for the error banner to display at the top of the page. Otherwise, the error banner gets hidden behind the top bar.